apps/sms/views/conversation/js/conversation.js

```diff
@@ -747,6 +752,7 @@ var ConversationView = {
     // TODO add the activity/forward/draft stuff here
     // instead of in afterEnter: Bug 1010223
 
+    // TODO: Will need to replace with Conversation service method.
```

Comment:
Ok, I think the question here would be: do we still need to cache currentId in conversation service? I think we can remove the dependency in Composer, but not sure about the activity part. If we still need currentId cross the view, it might be in conversation service and we'll need a method to cache it.